compressed_image_transport/cfg/CompressedSubscriber.cfg

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env python
+
+PACKAGE='compressed_image_transport'
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+
+mode_enum = gen.enum( [gen.const("unchanged", str_t, "unchanged", "keep image encoding"),
```

Comment:
OK, latest commit uses the constants, and I've also added the ones in the publisher by the same token.